### PR TITLE
Update translation readme jp

### PR DESCRIPTION
**Pull request by RRFHOUDEN**
#312

### CODE REVIEW

**Comment by chenrui333** [comment]
cc @Atena1118

**Comment by GindaChen** [comment]
@RRFHOUDEN 誠に、ありがとうございます！

**Comment by GindaChen** [comment]
@chenrui333 I will review the content.

File: README_JP.md [hunk 1]
```diff
@@ -119,4 +119,4 @@
 
 クリックしてください[shimoファイル](https://shimo.im/docs/DdWvXvtvpxrqrJ83)
 
-翻訳者 Sakula Air, Yoki, 校正者 Yoki W.
+翻訳者 Sakula Air, Yoki 校正者 Yoki W, [RRF](https://github.com/RRFHOUDEN)
```
Comment (GindaChen) [inline README_JP.md]: Good habit to add yourself as contributor!

**Comment by GindaChen** [comment]
@chenrui333 Would you like to give me the write permission of the repo?

**Comment by chenrui333** [comment]
@GindaChen today's SU topic. 😄 Frank would be the POC.

**Comment by GindaChen** [comment]
I see (but what is SU and POC again?

**Comment by chenrui333** [comment]
SU = standup, POC = point of contact :)

**Pull request merged**